core/src/backends/eclair/rest/node.rs

```diff
@@ -14,22 +14,19 @@ pub struct EclairRest {
 }
 
 impl EclairRest {
-    pub fn new(config: NodeConfig) -> Result<Self, Error> {
-        let config: EclairRestConfig = config.into();
-
-        let encoded_auth = format!("{}:{}", config.username, config.password);
+    pub fn new(config: EclairRestConfig) -> Result<Self, Error> {
+        let encoded_auth = format!("{}:{}", &config.username, &config.password);
```

Comment:
```suggestion
        let auth = format!("{}:{}", &config.username, &config.password);
```
The auth string isn't really encoded yet, so not convinced about the var name. Not the scope of this PR tho.